.github/workflows/ci.yml

```diff
@@ -172,7 +172,9 @@ jobs:
 
       - name: Install Tor [Ubuntu]
         if: ${{ contains(matrix.os, 'ubuntu') }}
-        run: sudo apt install tor
+        run: |
+          sudo add-apt-repository universe
+          sudo apt install tor
```

Comment:
you almost certainly want Tor's repository not this https://support.torproject.org/apt/#apt_tor-ubuntu

Reply:
Thanks for the tip!  The torproject has this [dozen steps installation guide](https://support.torproject.org/apt/tor-deb-repo/) where I first have to install wget or curl and gnupg etc...

This is for local-loopback testing for compatibility with Tor.   Should I really put half a screenful of Tor installation procedure (and that's only one operating system out of three) in the CI workflow config to get the latest minor revision?  Ubuntu 24.04 isn't _that_ old.

Reply:
Wasn't too bad after all, added installing from Tor in c5f108c2ccde201fb189a64ae0530b70440ac90b and 2c837e5cf90c5bd7c3923f5b97dbbc2ac02f7604.